core/blockchain.go

```diff
@@ -573,6 +586,25 @@ func (bc *BlockChain) SetHead(head uint64) error {
 	return err
 }
 
+func (bc *BlockChain) tryRewindBadBlocks() {
+	bc.chainmu.Lock()
+	defer bc.chainmu.Unlock()
+	block := bc.CurrentBlock()
+	snaps := bc.snaps
+	// Verified and Result is false
+	if snaps != nil && snaps.Snapshot(block.Root()) != nil &&
+		snaps.Snapshot(block.Root()).Verified() && !snaps.Snapshot(block.Root()).WaitVerified() {
```

Comment:
It costs me a while to figure out the difference between `Verified` and `WaitVerified` at the first glance. Since `Verified` returns the status of the verified process and `WaitVerified` returns the result of the verified process. But they share the same word `Verified`, can we have better names for these functions.

Reply:
sure, changed. Please check latest commit.